From: Gerd Hoffmann <kraxel@redhat.com>
Date: Thu, 13 Mar 2014 08:08:41 +0100
Subject: [PATCH] pick up any display device, not only vga

---
 OvmfPkg/Library/PlatformBdsLib/BdsPlatform.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/OvmfPkg/Library/PlatformBdsLib/BdsPlatform.c b/OvmfPkg/Library/PlatformBdsLib/BdsPlatform.c
index 0bc02ba..5024caf 100644
--- a/OvmfPkg/Library/PlatformBdsLib/BdsPlatform.c
+++ b/OvmfPkg/Library/PlatformBdsLib/BdsPlatform.c
@@ -586,7 +586,7 @@ DetectAndPreparePlatformPciDevicePath (
   //
   // Here we decide which VGA device to enable in PCI bus
   //
-  if (IS_PCI_VGA (Pci)) {
+  if (IS_PCI_DISPLAY (Pci)) {
     //
     // Add them to ConOut.
     //
